### PR TITLE
Fix ShowMore, Dismiss buttons styles in MessageBar (#1753)

### LINKED ISSUE #1753: ShowMore and dismiss buttons problem in BitMessageBar
In `BitMessageBar` component when `Truncated` and `OnDismiss` properties both got value then the related buttons get so close to each other and makes mobile user to click on show more button

| **Current State** | **Expected State** |
|--------|--------|
|    image    |    image    |

### PR DESCRIPTION
**Pull request by RezaKargar**
This closes #1753

### CODE REVIEW

**Comment by msynk** [comment]
@maryamhdr 
review plz

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/TextField/BitTextField.Fluent.scss [hunk 1]
```diff
@@ -57,6 +57,16 @@
...
     font-family: "Segoe UI", "Segoe UI Web (West European)", "Segoe UI", -apple-system, BlinkMacSystemFont, Roboto, "Helvetica Neue", sans-serif;
+
+    .bit-icon {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        width: rem(32px);
+        height: 100%;
+        font-size: rem(16);
+        color: $Grey160;
```
Comment (maryamhdr) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/TextField/BitTextField.Fluent.scss:68]: This PR is about the `BitMessageBar` component.
Why do we have some changes in this file?
@RezaKargar

Comment (RezaKargar) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/TextField/BitTextField.Fluent.scss:68]: The styles are for **bit-icon** inside the **BitTextField** but it was not wrapped inside the corresponding css selector so they were effecting the whole bit-icons even the ones inside **MessageBar**

Comment (maryamhdr) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/TextField/BitTextField.Fluent.scss:68]: > The styles are for **bit-icon** inside the **BitTextField** but it was not wrapped inside the corresponding css selector so they were effecting the whole bit-icons even the ones inside **MessageBar**

I got it...Thank you :)

**Review by maryamhdr — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**